### PR TITLE
LateNight PaleMoon: Improve TrackMenu and CueMenuPopup styles

### PR DESCRIPTION
**Pull request by Holzhaus**
I think the menus don't really fit well right now, I think this is much more consistent with the rest of the skin. What do you think?

### CODE REVIEW

File: res/skins/LateNight/style_palemoon.qss [hunk 1]
```diff
@@ -2995,3 +2999,22 @@ WEffectSelector,
     }
 /************** common styles for WEffectSelector ******************************
 *************** QSpinBox, QMenu, QToolTip *************************************/
+
+WColorPicker QPushButton,
+WCueMenuPopup #CueDeleteButton {
+  outline: none;
+  border-width: 2px 2px 2px 1px;
+  border-image: url(skin:/palemoon/buttons/btn_embedded_grid.svg) 2 2 2 1;
```
Comment (ronso0) [inline res/skins/LateNight/style_palemoon.qss]:
```suggestion
  border-image: url(skin:/palemoon/buttons/btn_embedded_library.svg) 2 2 2 1;
```
`_grid` has no border at the left.

File: res/skins/LateNight/style_palemoon.qss [hunk 2]
```diff
@@ -2995,3 +2999,22 @@ WEffectSelector,
...
+}
+
+WCueMenuPopup #CueDeleteButton:pressed {
+  background-color: #b24c12;
+  outline: none;
+  border-width: 2px 2px 2px 1px;
+  border-image: url(skin:/palemoon/buttons/btn_embedded_grid_active.svg) 2 2 2 1;
```
Comment (ronso0) [inline res/skins/LateNight/style_palemoon.qss]:
```suggestion
  border-image: url(skin:/palemoon/buttons/btn_embedded_library_active.svg) 2 2 2 1;
```

**Comment by ronso0** [comment]
For the menus in decks and spinboxes I apreciate the change.
In the library the menu blends in too much IMO, and the bottom border is almost invisible:
image

Different borders for menus in the library vs in the decks could help, but then we'd have to apply that also to tooltips in vrious places. I doubt the effort is worth it, I'd rather keep the plain style for popups.
But the WCueMenu button style is nice, also that of CueLabelEdit (except the black groove around it).

btw: inactive menu items are not greyed out properly, maybe you can fix that while you're at it:
image
And I wonder why active icons are darker than inactive ones.

**Comment by Holzhaus** [comment]
I improved the contrast and fixed the disabled menu items, please have another look:
Screenshot from 2020-08-23 15-52-48

File: res/skins/LateNight/style_palemoon.qss [hunk 3]
```diff
@@ -2794,7 +2801,26 @@ WEffectSelector::item,
...
+  #LibraryContainer QMenu QCheckBox:disabled,
+  WTrackMenu QCheckBox:disabled,
+  WTrackMenu QCheckBox:focus,
+  WTrackMenu QCheckBox:hover,
+  WTrackMenu QMenu QCheckBox:disabled,
+  #SkinSettingsButton[hover="true"],
+  #DeckSizeButton[hover="true"],
+  #SkinSettingsMixerToggle[hover="true"],
+  #SkinSettingsLabelButton[hover="true"] {
+    color: #777;
```
Comment (ronso0) [inline res/skins/LateNight/style_palemoon.qss:2822]: let's make this #555, and remove all selectors that are not `...:disabled` as it renders some hovered buttons unreadable.

File: res/skins/LateNight/style_palemoon.qss [hunk 4]
```diff
@@ -2995,3 +3021,22 @@ WEffectSelector,
     }
 /************** common styles for WEffectSelector ******************************
 *************** QSpinBox, QMenu, QToolTip *************************************/
+
+WColorPicker QPushButton,
+WCueMenuPopup #CueDeleteButton {
+  outline: none;
+  border-width: 2px 2px 2px 1px;
+  border-image: url(skin:/palemoon/buttons/btn_embedded_library.svg) 2 2 2 1;
```
Comment (ronso0) [inline res/skins/LateNight/style_palemoon.qss:3029]:
```suggestion
  border-width: 2px 2px 2px 2px;
  border-image: url(skin:/palemoon/buttons/btn_embedded_library.svg) 2 2 2 2;
```

File: res/skins/LateNight/style_palemoon.qss [hunk 5]
```diff
@@ -2995,3 +3021,22 @@ WEffectSelector,
...
+}
+
+WCueMenuPopup #CueDeleteButton:pressed {
+  background-color: #b24c12;
+  outline: none;
+  border-width: 2px 2px 2px 1px;
+  border-image: url(skin:/palemoon/buttons/btn_embedded_library_active.svg) 2 2 2 1;
```
Comment (ronso0) [inline res/skins/LateNight/style_palemoon.qss:3036]:
```suggestion
  border-width: 2px 2px 2px 2px;
  border-image: url(skin:/palemoon/buttons/btn_embedded_library_active.svg) 2 2 2 2;
```

**Comment by ronso0** [comment]
In general I'd rather stick with a constant border for all popups; right now it looks broken with the almost invisble borders at the right/bottom.

The only way to style the CueMenu color buttons somewhat like those in DlgAutoDJ or QHeaderView or any WPushButton is to use a dedicated btn_colorpicker image. For the label editor I'd pick the recessed style of OverviewBox.
(still trying to figure why the checked button is smaller)
image

**Comment by ronso0** [comment]
Even though I started a review, I think it makes more sense to adopt your proposals individually and add the button images myself. Reviewing and applying the changes back and forth etc. takes much longer than discussing some screenshots.

I'll post some further experiments here and I'd appreciate if we continue discussing those, so I'll keep this PR open.

**Comment by ronso0** [comment]
A minor bug with the color picker in the library track menu when overwriting the Fusion style:
on first invocation it looks oversized, after that the size adjusted to the button grid.
image

**Comment by ronso0** [comment]
I picked your changes and tweaked them a bit, like the constant border.
Please keep testing and let me know if it works for you. Thx!

**Pull request merged**